sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/ObjectAnchorsConversionClient.cs

```diff
@@ -25,7 +25,9 @@ public class ObjectAnchorsConversionClient
         /// <summary>
         /// The list of supported asset file types
         /// </summary>
-        public IEnumerable<AssetFileType> SupportedAssetFileTypes { get; }
+        public IReadOnlyList<AssetFileType> SupportedAssetFileTypes { get { return SupportedAssetFileTypesSet.ToList(); } }
```

Comment:
Can this be cached rather than converted each time?

Reply:
Cached